dj_rest_auth/urls.py

```diff
@@ -16,3 +17,13 @@
     url(r'^password/change/$', PasswordChangeView.as_view(),
         name='rest_password_change'),
 ]
+
+if getattr(settings, 'REST_USE_JWT', True):
```

Comment:
I think it would be better to have the default set to `False`:
```suggestion
if getattr(settings, 'REST_USE_JWT', False):
```
since JWT is optional.